### DISCUSSION TITLE
Is slate-slack active and can I get an invite?

### DISCUSSION THREAD

**Discussion by CAYdenberg** [ianstormtaylor/slate#5332] [General]
Hi,

Is the Slate Slack group active? The invitation bot seems to be broken: https://slate-slack.herokuapp.com/

Could I get an invite if so?

Casey

**Comment by dylans** [comment]
https://join.slack.com/t/slate-js/shared_invite/zt-1dfrsfubm-a7KoUp57oyKn86pZ67XrBQ
